design-system/react/src/components/Switch/styles.ts

```diff
@@ -2,16 +2,19 @@ import { css } from '@fuel-ui/css';
 
 export const root = css({
   all: 'unset',
+  layer: 'input-base',
   position: 'relative',
   width: 42,
   height: 25,
-  backgroundColor: '$inputBg',
   borderRadius: '$full',
-  boxShadow: '$sm',
   WebkitTapHighlightColor: 'rgba(0, 0, 0, 0)',
 
   '&:focus-visible': {
-    outline: '2px solid $gray5',
+    outline: '2px solid $intentsBase4',
+  },
+
+  '& .fuel_Icon': {
```

Comment:
```suggestion
  '& .fuel_icon': {
```

Reply:
hm, why this recommendation? Because the pattern in like `fuel_{Component}`, if you check, the Icon classname it's getting generated by a function

Reply:
my bad then sorry